### PR TITLE
Smartphone grammar/word choice fix

### PR DESCRIPTION
**Pull request by slater**
#### Summary
None

#### Purpose of change
Poor grammar/word choice.

#### Describe the solution
Better choice of grammar and words!

#### Describe alternatives you've considered
None.

### CODE REVIEW

**Comment by J-Fields** [comment]
- Double spaces should remain (refer to https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md)
- I believe the comma added after "camera" is incorrect since it doesn't split two independent clauses
- Elsewhere, I see lots of references to "Unified Power System" and some to "Unified Power Supply"... I'm not sure which is correct (and we should presumably make it consistent), but I don't think "Unified Power Supplies" is right

Other changes look good to me

**Comment by slater** [comment]
whelp, think i messed up the commit. not sure how to remove the wrong ones :(

**Review by NetSysFire — APPROVED** [review]

**Comment by NetSysFire** [comment]
You do not need to. This commit will most likely be squashed anyways.

**Comment by slater** [comment]
> You do not need to. This PR will most likely be squashed anyways.

Any specific reason?

**Comment by NetSysFire** [comment]
The reason is because it will be squashed. That means all of the commits become one single big commit.

And you need to fix the summary of this PR.

**Comment by slater** [comment]
> The reason is because it will be squashed. That means all of the commits become one single big commit.
> 
> And you need to fix the summary of this PR.

Got it. I thought you meant squashed as in, rejected. I updated the summary. Thanks!

**Comment by NetSysFire** [comment]
Please do read https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/CONTRIBUTING.md#all-prs-should-have-a-summary-section-with-one-line

File: data/json/items/tool/electronics.json [hunk 1]
```diff
@@ -495,7 +495,7 @@
     "id": "smart_phone",
     "type": "TOOL",
     "name": { "str": "smartphone" },
-    "description": "A popular, fancy smartphone.  Capable of making photos due to integrated camera and illuminating an area as per flashlight app, assuming it has enough charge.  The smartphone also has a clock app that includes an alarm and an app for tracking consumed calories.  Runs on a small, rechargeable power cell compatible with Unified Power Supply.",
+    "description": "A popular, fancy smartphone.  Capable of taking photos with its integrated camera and illuminating an area with its flashlight app, assuming it has enough charge.  The smartphone also has a clock app that includes an alarm and an app for tracking consumed calories. Runs on a small, rechargeable power cell compatible with a Unified Power Supply.",
```
Comment (kevingranade) [inline data/json/items/tool/electronics.json:498]:
```suggestion
    "description": "A popular, fancy smartphone.  Capable of taking photos with its integrated camera and illuminating an area with its flashlight app, assuming it has enough charge.  The smartphone also has a clock app that includes an alarm and an app for tracking consumed calories.  Runs on a small, rechargeable power cell compatible with a Unified Power Supply.",
```

**Pull request merged**